app/src/main/graphql/fragments.graphql

```diff
@@ -310,6 +310,11 @@ fragment reward on Reward {
     startsAt
     endsAt
     rewardType
+    allowedAddons {
```

Comment:
Backing -> reward -> allowedAddons was never queried, therefore reward.hasAddOns was always false for the backed reward